core/util.py

```diff
@@ -129,6 +131,100 @@ def load_example_data():
         table_df = pd.read_csv(resources / f"{table}.csv")
         if table == "project_dim":
             table_df["postcodes"] = table_df["postcodes"].str.split(",")
+        if table == "project_progress":
+            table_df = table_df.replace(np.nan, None)
```

Comment:
this line is repeated just prior to every time the `move_event_data_to_json_blob `function is called, so can it just be moved insidethe function to avoid this duplication?

Reply:
Moved inside which function?

Reply:
The issue is it's better to have the ```cols_to_jsonb``` instantiated inside the mapping for the sake of clarity and consistency, but we can't call the mapping here because it creates a circular import if we import it into ```util```.

Reply:
Although perhaps this function can go in ```conftest``` as that seems somewhat more appropriate.

Reply:
But that would also create a circular import...

Reply:
to first line of `move_data_to_json_blov`

Reply:
I'm specifically talking about the line 
`table_df = table_df.replace(np.nan, None)`

Reply:
Ah ok, got it